.github/workflows/Build_and_publish_package.yml

```diff
@@ -0,0 +1,33 @@
+name: Build and Deploy Packages
+
+on: [push]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+jobs:
+  build:
+    # The CMake configure and build commands are platform agnostic and should work equally
+    # well on Windows or Mac.  You can convert this to a matrix build if you need
+    # cross-platform coverage.
+    # See: https://docs.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#configuring-a-build-matrix
+    runs-on: windows-latest
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # avoid shallow clone so nbgv can do its work.
+    # Add tool nbgv for build support
+    - name: Update Gitversioning
+      run: |
+        dotnet tool install --tool-path . nbgv
+        .\nbgv cloud
```

Comment:
Can we check if we could use their action instead? https://github.com/marketplace/actions/nerdbank-gitversioning

Reply:
Changed to using the github action